src/index.js

```diff
@@ -72,6 +73,16 @@ const userIsAdminOrEditor = connectedReduxRedirect({
     allowRedirectBack: false
 });
 
+const userIsAdmin = connectedReduxRedirect({
```

Comment:
can't see it is used in your code, you used `userIsAdminOrEditor`

Reply:
Cheers, yes this was added and used then I removed the use of it and forgot to get rid of the function. But, since making this PR seems the permissions-api only allows Admins now to create teams so will be switching it back to use this function.